### PR TITLE
Merge some commits from cardano-sl-0.5-rc to master

### PR DESCRIPTION
**Pull request by gromakovsky**

### CODE REVIEW

File: core/Pos/Binary/Crypto.hs [hunk 1]
```diff
@@ -142,7 +142,7 @@ secretKeyLength, publicKeyLength, signatureLength, chainCodeLength,
     encryptedKeyLength, passphraseLength :: Int
 secretKeyLength = 32
 publicKeyLength = 32
-encryptedKeyLength = 96
+encryptedKeyLength = 128
```
Comment (akegalj) [inline core/Pos/Binary/Crypto.hs]: we use `encryptedKeyLength = 96` in `cardano-sl-0.5-rc`. This change was reverted here https://github.com/input-output-hk/cardano-sl/pull/1086

Comment (akegalj) [inline core/Pos/Binary/Crypto.hs]: @neongreen which constants do we want in `master` ? I guess its expected to have these change in `master`?

Comment (neongreen) [inline core/Pos/Binary/Crypto.hs]: It should be in master, yep

**Comment by neongreen** [comment]
> fatal: Remote branch merge/misha-asked-to-have-┬½merge/┬╗-prefix not found in upstream origin

Silly Windows.

**Comment by neongreen** [comment]
Merged and updated YT.

**Pull request merged**